templates/images-template.yaml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: openstack.k-orc.cloud/v1alpha1
+kind: Image
+metadata:
+  name: node-image
+spec:
+  managementPolicy: managed
+  resource:
+    name: flatcar_production
+    content:
+      diskFormat: qcow2
+      download:
+        url: ${NODE_IMAGE_URL:="https://stable.release.flatcar-linux.net/amd64-usr/current/flatcar_production_openstack_image.img"}
+  cloudCredentialsRef:
+    secretName: dev-test-cloud-config
+    cloudName: capo-e2e
+---
+apiVersion: openstack.k-orc.cloud/v1alpha1
+kind: Image
+metadata:
+  name: bastion-image
+spec:
+  managementPolicy: managed
+  resource:
+    name: ubuntu-22.04
+    content:
+      diskFormat: qcow2
+      download:
+        url: ${BASTION_IMAGE_URL:="https://cloud-images.ubuntu.com/releases/jammy/release/ubuntu-22.04-server-cloudimg-amd64.img"}
```

Comment:
nit: to be consistent with other ubuntu images, we could have given the link of 24.04.